src/converter/symbol.py

```diff
@@ -26,19 +27,12 @@ def convert(fig_symbol):
         symbolID=utils.gen_object_id(fig_symbol["guid"]),
     )
 
+    if utils.has_auto_layout(fig_symbol):
+        master = convert_auto_layout(master, fig_symbol)
+
     # Keep the base ID as the symbol reference, create a new one for the container
     master.do_objectID = utils.gen_object_id(fig_symbol["guid"], b"symbol_master")
 
-    # Also add group layout if auto-layout is enabled
-    axis = LAYOUT_AXIS[fig_symbol.get("stackMode", "NONE")]
-    if axis is not None:
-        anchor = LAYOUT_ANCHOR[fig_symbol.get("stackPrimaryAlignItems", "MIN")]
-
-        master.groupLayout = InferredGroupLayout(
-            axis=axis,
-            layoutAnchor=anchor,
-        )
```

Comment:
I believe this was using Smart Layout if auto layout was being used. We don't need to do this now that we have stacks 🎉